app/src/full/java/org/openobservatory/ooniprobe/common/FirebaseMessagingService.java

```diff
@@ -17,7 +21,15 @@ public void onMessageReceived(RemoteMessage remoteMessage) {
 		// decode message data and extract meaningful information from it: title, body, badge, etc.
 		CountlyPush.Message message = CountlyPush.decodeMessage(remoteMessage.getData());
 
-		Boolean result = CountlyPush.displayMessage(getApplicationContext(), message, R.drawable.notification_icon, null);
+		Intent notificationIntent = null;
+		if (!message.has("c.l")) {
```

Comment:
Wow, `c.l` is very cryptic! Is this data format documented in some place?

Reply:
in countly doc: https://support.count.ly/hc/en-us/articles/360037754031-Android
Search for "c.l"